.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,38 @@
+name: "Feature Request"
+description: Create a new ticket for a new feature request
+title: "[REQUEST]: "
+labels: ["enhancement", "triage"]
+
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: "Summary"
+      description: Provide a brief explanation of the feature
+      placeholder: Describe in a few lines your feature request
+    validations:
+      required: true
+  - type: textarea
+    id: basic_example
+    attributes:
+      label: "Basic Example"
+      description: Provide an example or two (User Story).
+      placeholder: A few specific words about your feature request.
+    validations:
+      required: true
+  - type: textarea
+    id: tradeoffs
+    attributes:
+      label: "Trade-offs"
+      description: What are the impacts of your feature request?
+      placeholder: Identify the drawbacks and impacts while being neutral on your feature request
+    validations:
+      required: true
+  - type: textarea
+    id: unresolved_questions
+    attributes:
+      label: "Unresolved questions"
+      description: What questions still remain unresolved?
```

Comment:
I get what this is going for, but it could be clarified a little bit. Something along the lines of "If you created this feature request in response to an problem you encountered, what parts of that problem does this feature not address?".

Reply:
I like that! I'll put that in there instead.